.gitlab/common/macos.yml

```
@@ -66,6 +66,19 @@
   - rm -f dda.tar.gz
   - export PATH="$DDA_DIR:$PATH"
   - dda self dep sync -f legacy-tasks
+  # Install awscli
+  - export AWSCLI_DIR="$TMPDIR/awscli-${CI_JOB_ID}"
+  - |
+    echo 'Installing AWS CLI...'
+    AWSCLI_VERSION=1.29.45
+    awscli_d="$(mktemp -d)"
+    pushd "$awscli_d"
+    curl -fsSL "https://s3.amazonaws.com/aws-cli/awscli-bundle-$AWSCLI_VERSION.zip" --output bundle.zip
+    unzip bundle.zip
+    sudo /usr/local/bin/python"$PYTHON_VERSION" awscli-bundle/install -i $AWSCLI_DIR/installation -b $AWSCLI_DIR/aws
```

I was wrong in my initial draft (now [fixed](https://github.com/DataDog/ci-platform-machine-images/pull/393/files#diff-4aba17db6f4285f53c180fa906edc6a71758c8ec3423dd703a4f63f28e05cf3cR113)): the PYTHON_VERSION here denotes the `venv` one, whereas we should use the `system` one:
```suggestion
    sudo awscli-bundle/install -i $AWSCLI_DIR/installation -b $AWSCLI_DIR/aws
```